examples/pytorch/pinsage/data_utils.py

```diff
@@ -0,0 +1,77 @@
+import torch
+import dgl
+import numpy as np
+import scipy.sparse as ssp
+
+def train_test_split_by_time(g, column, etype, itype):
```

Comment:
Can you give some comments about why split data this way?

Reply:
I almost forgot about this PR...  Done.